src/cmd-buildextend-metal

```diff
@@ -143,6 +143,17 @@ if [ "${image_type}" = dasd ] || [ "${image_type}" = metal4k ]; then
     ignition_platform_id=metal
 fi
 
+# Firstboot kernel arguments.
+firstboot_kargs="$(python3 -c '
+import sys, yaml;
```

Comment:
Minor/optional: I think we can drop the semicolons here and below since this is a proper multiline string now?